integr8ly/integreatly-operator
# [INTLY-3093] Update 3scale reconciler to use high availability mode with CRO

### Description 
Use the `cloud-resource-operator` to deploy AWS resources in managed environments for 3scale

### Verification
- Run the integreatly operator from this branch 
- Run the cloud resource operator
- Create a new `installation` resource:
```yml
apiVersion: integreatly.org/v1alpha1
kind: Installation
metadata:
  name: example-installation
spec:
  type: managed
  namespacePrefix: integreatly-
  routingSubdomain: example.com
  masterUrl: http://master.example.com
  selfSignedCerts: true
  useExternalResources: true
```
- Create a VPC peering connection and update the two VPC route tables to point to the peering connection 
- Verify that all 3scale pods are running 
- Verify that two elasticache instances were created and are available and one RDS instance was created and is available 
- Verify that normal 3scale actions can be performed (inviting a user, creating a service, etc)
